test/features/discord.feature

```diff
@@ -6,7 +6,8 @@ Feature: discord module
     And the response should be "[]"
 
   Scenario: add a new user
-    Given make a POST request to "/discord" with:
+    Given authorisation
+    And make a POST request to "/discord" with:
```

Comment:
We should probably have 2 more tests:

- get list of users with authorisation token and check it still works
- add a new user without authorisation token checking it fails correctly